extensions/amp-next-page/0.1/test/test-amp-next-page.js

```diff
@@ -0,0 +1,127 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {AmpNextPage} from '../amp-next-page';
+import {Services} from '../../../../src/services';
+import {layoutRectLtwh} from '../../../../src/layout-rect';
+import {macroTask} from '../../../../testing/yield';
+import {toggleExperiment} from '../../../../src/experiments';
+
+describes.realWin('amp-next-page component', {
+  amp: {
+    extensions: ['amp-next-page'],
+  },
+},
+env => {
+  let win, doc, ampdoc;
+  let element;
+  let nextPage;
+  let xhrMock;
+  let viewport;
+  let sizes;
+
+  beforeEach(() => {
+    win = env.win;
+    doc = win.document;
+    ampdoc = env.ampdoc;
+    viewport = Services.viewportForDoc(ampdoc);
+    sizes = viewport.getSize();
+
+    toggleExperiment(win, 'amp-next-page', true);
```

Comment:
you can move this to "experiments" as part of the JSON config ( just after "extensions")

Reply:
You can only specify `experiments` on `describes.integration()`, not `describes.realWin()`.